src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -13,12 +13,17 @@ import {
 import { Transition, animated } from 'react-spring/renderprops'
 import { useEsc } from '../../hooks/useKeyboardArrows'
 import Network from './Network/Network'
+import NotificationsManager from './Notifications/NotificationsManager'
 
-const SECTIONS = new Map([['network', 'Network']])
+const SECTIONS = new Map([
+  ['network', 'Network'],
+  ['notifications', 'Email Notifications'],
```

Comment:
```suggestion
  ['notifications', 'Notifications'],
```

Not sure, but should we update this name to be consistent with the client?